### ISSUE TITLE
Arcane contribution mage talent proc

### ISSUE THREAD

**Issue by Linniee** [elysium-project/server#1868]
Hi, 
the talent Arcane contribution which could be found on arcane talent tree, does not work properly. 

How it should work:
Gives you a 3/6/10% chance of entering a Clearcasting state after any damage spell **hits** a target. The Clearcasting state reduces the mana cost of your next damage spell by 100%.

How it works:
The Clearcasting state procs after casting any damaging spell no matter if it hitted any target. For example you can stand alone and cast frost nova, no enemy around, and the state would proc. And you can cast blizzard on hundred of mobs but you have only chance to proc it once when you started to cast it, not for each successful hit.

**Comment by truefireboykoko** [comment]
fix plz, want it to proc from my arcane missiles and explosions, impossible to enjoy playing arcane without that tallent working

**Comment by Phoosy** [comment]
Reference: https://github.com/elysium-project/server/pull/645

**Comment by corrupteddrake** [comment]
Yes its too bad that this is still a problem.  I submitted a bug ticket to them the very week that Elysium came out, and they have yet to fix the problem.

**Comment by thepparker** [comment]
Just to be clear, the confirmed issue here is that free casting and not hitting anything procs Clearcasting, NOT that Blizzard does not have a chance per tick. AM and Blizzard do NOT have a chance to proc each tick.

**Comment by truefireboykoko** [comment]
are you dumb thepparker? how the fk aoe knows what foe will get crit? you say its impossible to calculate each tick? get real. arcane missiles is aura that casts every second a spell that is actualy a damage spell that is... ffs its so obvious, and even if initial cast triggers gkd, even if its waits 1 sec till first shot, the cleancasting is bigger problem, or is its more like about stupid administration and retarded programmers?

**Comment by thepparker** [comment]
Nope, just interested in Blizzlike behaviour. https://forum.nostalrius.org/viewtopic.php?f=45&t=3373
If it were 10% per hit, players like this would get a proc every single spell cast https://www.youtube.com/watch?v=M8E6-mZrU4I

Try doing some research next time instead of claiming that something should work a specific way because you *think it should*.

**Comment by truefireboykoko** [comment]
so mr... whatever you are. the video you link with frost mage with frost build which is not include arcane conc and there is no need for that tbh

but it is not the point, you saying that http://www.wowhead.com/spell=245285/blizzard
Persistent Area Aura: Periodic Damage
Value: 1250 every 2 seconds
Radius: 8 yards

work same way as this http://www.wowhead.com/spell=209856/arcane-missiles
Apply Aura: Periodically trigger spell
Interval: 1 second 

try to do some brain work before declaring that you understand

**Comment by thepparker** [comment]
https://www.youtube.com/watch?v=3O_pNDc73MM&feature=youtu.be&t=819

I can link sources all day man. The spell effect mechanics are irrelevant.

**Comment by truefireboykoko** [comment]
https://youtu.be/r9S8uPZ-4tY?t=1m

i can link all day too man.

**Comment by thepparker** [comment]
I lol'd. Anyway, enough shitting up this issue report.

**Comment by truefireboykoko** [comment]
yea so levae for the good, and stop shitting with your worthless behaviour

**Comment by Phoosy** [comment]
You seemed to miss the fact that this project is ensuring that mechanics respect _vanilla_ era behavior as best as possible. The video you linked is Cataclysm content and is therefore not valid. As he stated, you'll need to find evidence that meets this criteria.

**Comment by truefireboykoko** [comment]
actualy, video that i linked about ending of wrath period when testing servers for cata were up, its just about you to see how cleancast may proc on random missile(in case of wrath when that tallent(arcane concentration) were still existed) not on starting of aura. each missile of aura(arcane missiles) is counts as a damageing spell that is deal damage and may trigger cleancasting when the spell itself(arcane missiles) counts as damageing spell that deals no damage by itself but arcane missile trigger and requires mana that may be redused by cleancast 

in case of blizzard(spell) as i post before 
http://www.wowhead.com/spell=245285/blizzard
Persistent Area Aura: Periodic Damage
Value: 1250 every 2 seconds
Radius: 8 yards

works not the same way as this http://www.wowhead.com/spell=209856/arcane-missiles
Apply Aura: Periodically trigger spell
Interval: 1 second

blizzard counts as a periodic damage like any dot in the game.
missile itself is a damageing spells that is should be triggered for every second that aura is maintaining, therefore whole question about "argument really comes down to if you have higher than elementary school reading comprehension skills or not." seems to be more likely applicable to you mr..."inedmyproof"

in case of other aoe's arcane concentration might be just disabled to proc after some abuses, pretty common policy for blizzard(company) or maybe it requires direct hit 

p.s. i dont have to do anything for you for free(in case if you so young and insecure to believe, that all owe to you some thing)
